### ISSUE TITLE
DS4Windows doesn't work with Forza Horizon 4, version 3.0.11

### ISSUE THREAD

**Issue by xavierhazzardadmin** [Jays2Kings/DS4Windows#694]
I am using the Sony Dual Sense PS5 controller on my PC. 

I used to be able to turn on the "Hide DS4 controller" option and be able to run Forza Horizon 4 with full controller support.

Now, firstly the latest version has taken that option away, the release notes don't really tell why. **Removed obsolete 'Hide DS4 Controller' option again. Can't have options**. 1. What is replacing this? 2. What does _"Can't have options mean?"_.

Secondly, when I re-download the 3.0.10 version, even if I have the "hide  DS4 controller" option on, Forza glitches where I can hold the buttons on the controller and the interface shows the XBOX controller button, but won't register any of the inputs apart from up and down for selecting continue, options and so on.

I am running Windows 11 preview but this issue was also persisting on Windows 10 Pro Version 21H1.

Please could you give some information if I am going about this wrong or if there is something that needs fixing. 
Many Thanks

**Comment by Drak00l** [comment]
Yeah same here, i need "Hide DS4 controller" back, some games like assassins creed, dont work correctly without that option, its like it detects at the same time keyboard and gamepad, or a duplicate gamepad.

Any solution? please take it back.
